lib/tasks/marks.rake

```diff
@@ -4,14 +4,18 @@ namespace :db do
   task :marks => :environment do
     puts 'Assign Marks for Assignments'
 
-    #Right now, only generate marks for two assignments
-    Grouping.where(assignment_id: [1, 2]).each do |grouping|
+    #Right now, only generate marks for three assignments
+    Grouping.where(assignment_id: [1, 2, 3]).each do |grouping|
       time = grouping.assignment.submission_rule.calculate_collection_time.localtime
       new_submission = Submission.create_by_timestamp(grouping, time)
       result = new_submission.results.first
       grouping.is_collected = true
       grouping.save
 
+      if grouping.assignment_id == 1
```

Comment:
Using ids is quite brittle. I recommend instead checking `grouping.assignment.short_identifier` or something.